e2e/tests/functional/moveAndLinkObjects.e2e.spec.js

```diff
@@ -137,6 +157,59 @@ test.describe('Move item tests', () => {
         let okButtonStateDisabled2 = await okButton2.isDisabled();
         expect(okButtonStateDisabled2).toBeTruthy();
     });
+
+    test('Create a basic object and verify that it can be linked to another folder', async ({ page, openmctConfig }) => {
+        const { myItemsFolderName } = openmctConfig;

```

Comment:
@scottbell @ozyx is there a way we could move this a higher scope?

Reply:
@unlikelyzero how do you mean?

Reply:
Could this be set for the entire suite instead of individual tests?

Reply:
There _should_ be a way to do this. The need for `myItemsFolderName` is likely unavoidable in some scenarios, but since it's a deployment-wide configuration we shouldn't have to import it every test. I'm working on a PR which will include improvements to tree selection, I'll see if I can improve our usage of this as a part of those changes. That's the most common scenario where we use this anyway.